rcl/src/rcl/node.c

```diff
@@ -179,7 +211,47 @@ rcl_node_init(
   }
   // actual domain id
   node->impl->actual_domain_id = domain_id;
-  node->impl->rmw_node_handle = rmw_create_node(name, local_namespace_, domain_id);
+
+  const char * ros_enable_security = NULL;
+  const char * ros_enforce_security = NULL;
+
+  if (rcutils_get_env(sros_enable_security_var_name, &ros_enable_security)) {
+    RCL_SET_ERROR_MSG("fail fetching environment variable", rcl_get_default_allocator());
```

Comment:
oh cool I didn't know about that. thanks @wjwwood for the pointer! https://github.com/ros2/rcl/pull/125/commits/1407fe7ab09c123ad045cbf6604fd5b2192ac0eb